11/umbraco-cms/reference/templating/modelsbuilder/understand-and-extend.md

```diff
@@ -117,9 +117,15 @@ Furthermore a generated model will always be instantiated with its default const
 
 For more complex partial classes, you'll have to use the full version of the [Models Builder](https://github.com/zpqrtbnk/Zbu.ModelsBuilder).
 
+### IModelsGenerator
+
+As of Umbraco 11.4, the IModelsGenerator interface has been added. If you want to customize how the models are generated, you can make your own implementation of the `IModelsGenerator` interface, and then overwrite the Umbraco implementation with dependency injection. 
```

Comment:
I've committed your suggestion @Zeegaan , but only to make another suggestion to that suggestion (🙃 ), to shorten the sentence a bit!
Let me know if it still works 🤞 

```suggestion
As of Umbraco 11.4, the IModelsGenerator interface has been added. If you want to customize how the models are generated, you can make your own implementation of the `IModelsGenerator` interface. You can then overwrite the Umbraco implementation with dependency injection. 
```

Reply:
🤣 
Looks good 💪